include/const.hpp

```diff
@@ -29,11 +30,14 @@ static constexpr auto everLcdDbusObj =
     "panel1";
 static constexpr auto everBMCObj =
     "/xyz/openbmc_project/inventory/system/chassis/motherboard/bmc";
+static constexpr auto bonelBootFailChipFRU = systemDbusObj;
```

Comment:
bonnell* spellcheck

Reply:
@GiridhariKrishna  As we discussed today, bonnellBootFailChipFRU is systemDbusObj thats right, but the place where you have fed this information is wrong. The boot fail PIC FRU details are in this map https://github.com/ibm-openbmc/ibm-panel/blob/1050/include/const.hpp#L80 . please update this map for bonnell.

and in this baseDataMap https://github.com/ibm-openbmc/ibm-panel/blob/1050/src/panel_app_main.cpp#L12
you need to provide bonnellBaseDbusObj path and not the bootFailPIC fru path.

Reply:
> bonnell* spellcheck

It's intentional

Reply:
that should be ok to give complete word - bonnell with right spelling. its your call.

Reply:
> and in this baseDataMap https://github.com/ibm-openbmc/ibm-panel/blob/1050/src/panel_app_main.cpp#L12
> you need to provide bonnellBaseDbusObj path and not the bootFailPIC fru path.

can you please address this comment?

Reply:
Yes, initially thought so but confused with systemDbusobj, will change that.

Reply:
Sorry, changed that already, will update.